src/com/alevel/module1/Lock.java

```diff
@@ -0,0 +1,20 @@
+package com.alevel.module1;
+
+public class Lock {
+    private final int[] mass;
+
+    public Lock(int[] mass) {
+        this.mass = mass;
+    }
+
+    public int[] getArray() {
+        return (mass == null) ? null : mass.clone();
+    }
+
+    public void printArray() {
+        for (int i : mass) {
+            System.out.printf("%8d", i);
+        }
+        System.out.println();
+    }
+}
```

Comment:
There is a way how to change the array with the current implementation

Reply:
`System.out.println("Lock array:");
        Lock lock = new Lock(mass);
        lock.getArray()[0] = 2;
        lock.printArray();`

These lines from the program demonstrate the inability to change the content

Reply:
This is one of two options for how you can change the array. There is one more.